vendor/github.com/jesseduffield/gocui/view.go

```diff
@@ -1348,8 +1348,8 @@ func (v *View) GetClickedTabIndex(x int) int {
 		return 0
 	}
 
-	charX := 1
-	if x <= charX {
+	charX := runewidth.StringWidth(v.Title)
+	if x <= 1 {
```

Comment:
This needs to stay `x <= charX`. We're testing whether we're to the left of the first tab here.